Assets/BossRoom/Scripts/Client/Game/State/ClientCharSelectState.cs

```diff
@@ -115,6 +115,15 @@ protected override void Awake()
             };
```

Comment:
After discussion, knowing workaround is pretty small, let's identify which lines are workarounds and waiting for which fix. (+jira ticket for it)

Reply:
jira ticket on our side to track this